docs/application/dotnet/guides/nui/xaml/layout-define-example.md

```diff
@@ -6,7 +6,7 @@ You can use XAML to define the layout of user interface in your applications. To
 3. Put the XAML file in the appropriate file directory
 
 > **Note**
-> 
+>
 > Tizen NUI XAML application platform provides a XAML application template and you can use XAML since **Tizen.NET API version 7**.
```

Comment:
Global.

The note style in Tizen document is updated. 

**Reference :** https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md#alerts-note-tip-important-caution-warning

So, would you please change the notes in this page and xaml-support-for-tizen-nui.md?

\> \[!NOTE]
\> Tizen NUI XAML application platform provides a XAML application template and you can use XAML since \*\*Tizen.NET API version 7**.

Reply:
Okay, Changed all notes to [!NOTE].